NeosModLoader/ModConfigurationDefinitionBuilder.cs

```diff
@@ -80,7 +80,7 @@ private void ProcessField(FieldInfo field)
             }
 
             ModConfigurationKey fieldValue = (ModConfigurationKey)field.GetValue(field.IsStatic ? null : Owner);
-            Logger.DebugInternal($"{Owner.Name} autoregistering config key: {field}");
+            //Logger.DebugInternal($"{Owner.Name} autoregistering config key: {field}");
```

Comment:
Please just go ahead and remove this completely. I'm not a fan of commented-out code.